src/shogun/ensemble/MeanRule.cpp

```diff
@@ -35,7 +36,7 @@ SGVector<float64_t> CMeanRule::combine(const SGMatrix<float64_t>& ensemble_resul
 	SGVector<float64_t> mean_labels(row_sum, ensemble_result.num_rows);
 
 	float64_t scale = 1/(float64_t)ensemble_result.num_cols;
-	mean_labels.scale(scale);
```

Comment:
@vigsterkr whats your suggestion here?
Ignore linalg, make a double implementation, or make this class unavailable for non c++11 ?